### ISSUE TITLE
Run tests against ScubaGear for WCAG 2.2 comformance 

### ISSUE THREAD

**Issue by mitchelbaker-cisa** [cisagov/ScubaGear#879]
# 💡 Summary #

Go through the report overview and product reports, then determine for each WCAG 2.2 guidelines if ScubaGear conforms to level A and AA success criteria. This is listed under the software requirements for 508 compliance. 

https://www.access-board.gov/ict/#E207-software
https://www.access-board.gov/ict/#chapter-5-software

From [E207.2 WCAG Comformance](https://www.access-board.gov/ict/#E207.2): 

"User interface components, as well as the content of platforms and applications, shall conform to Level A and Level AA Success Criteria and Conformance Requirements in WCAG 2.2 (incorporated by reference, see 702.10.1)."

WCAG 2.0 guidelines: https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2

## Motivation and context ##

This would be useful because we are looking to make HTML reporting improvements in an upcoming release, #820, and ScubaGear should be accessible to people of all backgrounds.

## Implementation notes ##

There are many [tools ](https://www.w3.org/WAI/ER/tools/) to aid with testing accessibility, but the majority of the WCAG 2.2 testing will have to be a case by case process for each guideline.

1. For example, `Guideline 1.1 - Text Alternatives`: 
a. Go through any non-text content in the reports. There should be a text alternative that serves the equivalent purpose. We already have alt attributes for images, but for screen readers it will be helpful to include aria-labels. 
2. It will help to create a checklist of sorts to mark which guidelines we've covered. Some of them such as `Guideline 1.2 - Time-based media` doesn't apply to ScubaGear. 

## Acceptance criteria ##

How do we know when this work is done?

- [x] Create a mapping of all WCAG 2.2 guidelines, document which guidelines we cover. If we do not support the guideline, then create issues and link them here.

**Comment by mitchelbaker-cisa** [comment]
| Guideline | Success Criteria | Description | Application in Report | Pass/Fail |  
| -------- | ------- | ------- | ------- | ------- |
| 1.1.1 Non-Text Content | A | All non-text content that is presented to the user has a text alternative that serves the equivalent purpose | CISA logo in <header>; add aria-label="Return to the report summary" | TBD |
| | | | | |
